changelog/8924.improvement.md

```diff
@@ -0,0 +1 @@
+Send user message tokens along with the `UserUttered` event through the event broker to Rasa X.
```

Comment:
Since we only send token indices and not values...
```suggestion
Send indices of user message tokens along with the `UserUttered` event through the event broker to Rasa X.
```

Reply:
Good catch! This was a relic of initial commits - some changes were decided upon just today 😄